packages/mdc-menu/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+@mixin mdc-menu-width($width) {
+  width: $width;
+}
+
+@mixin mdc-menu-width-calc($multiplier) {
```

Comment:
Maybe rename this to something like:

- `mdc-menu-width-multiplier($multiplier)`

Or, better yet, combine these two functions into one:

```sass
@mixin mdc-menu-width($width) {
  @if unit­les­s($­width) {
    width: $width * $mdc-menu-width-base;
  } @else {
    width: $width;
  }
}

...

@include mdc-menu-width(3); // multiplier
@include mdc-menu-width(300px); // verbatim
@include mdc-menu-width(calc(100% - 50px)); // verbatim
```